program/rust/src/tests/test_utils.rs

```diff
@@ -0,0 +1,64 @@
+use crate::c_oracle_header::PythAccount;
+use solana_program::account_info::AccountInfo;
+use solana_program::clock::Epoch;
+use solana_program::native_token::LAMPORTS_PER_SOL;
+use solana_program::pubkey::Pubkey;
+use solana_program::rent::Rent;
+use solana_program::system_program;
+
+const UPPER_BOUND_OF_ALL_ACCOUNT_SIZES: usize = 20536;
+
+/// The goal of this struct is to easily instantiate zeroed solana accounts
+/// for the Pyth program to use in tests.
+pub struct AccountSetup {
+    key:     Pubkey,
+    owner:   Pubkey,
+    balance: u64,
+    size:    usize,
+    data:    [u8; UPPER_BOUND_OF_ALL_ACCOUNT_SIZES],
```

Comment:
maybe leave a comment as to why this array needs to be allocated here (I think this is to get the ownership right?)